client/CMT.cpp

```diff
@@ -430,15 +430,15 @@ void playIntro()
 {
 	auto audioData = CCS->videoh->getAudio(VideoPath::builtin("3DOLOGO.SMK"));
 	int sound = CCS->soundh->playSound(audioData);
-	if(CCS->videoh->openAndPlayVideo(VideoPath::builtin("3DOLOGO.SMK"), 0, 1, true, true))
+	if(CCS->videoh->openAndPlayVideo(VideoPath::builtin("3DOLOGO.SMK"), 0, 1, true, true, false))
```

Comment:
that `true, true. false` part is anything but readable. Can probably be replaced with single bool (intro mode or page flip mode, don't think that there are other places that use this mode). Or with more clear enum with 2-3 values

Reply:
The campaign screen calls open() and update(), should that use openAndPlayVideo() as well?

Reply:
No. open() + update() is preferable to openAndPlay.

openAndPlay() effectively blocks all game activity other than video playback. Might be useful for intro, but not suitable when there are other elements (e.g mission into subtitles)

Meanwhile open() + update() work in line with our UI system, its only problem is global state - only one video can be played at once.